### PR TITLE
test: Mark TestStorageLvm2.testRaidRepair as "destructive"

### PR DESCRIPTION
**Pull request by mvollmer**
And make it produce evidence for our naughty matcher.

### CODE REVIEW

**Comment by martinpitt** [comment]
Good idea in principle, if tests weren't so red. Also, it seems https://bugzilla.redhat.com/show_bug.cgi?id=2256433 and https://bugzilla.redhat.com/show_bug.cgi?id=2256432 are duplicates? Let's close the latter one.

File: test/verify/check-storage-lvm2 [hunk 1]
```diff
@@ -900,5 +806,121 @@ class TestStorageLvm2(storagelib.StorageCase):
...
+class TestStorageLvm2Destructive(storagelib.StorageCase):
+
+    def can_do_layouts(self):
+        return self.storaged_version >= [2, 10]
+
+    def skip_if_no_layouts(self):
+        if not self.can_do_layouts():
+            raise unittest.SkipTest("raid layouts not supported")
+
+    # This is only "destuctive" because of
```
Comment (martinpitt) [inline test/verify/check-storage-lvm2:818]: destr**u**ctive

Most typos in comments don't matter, but that may be one which we want to grep for

**Comment by mvollmer** [comment]
> Also, it seems https://bugzilla.redhat.com/show_bug.cgi?id=2256433 and https://bugzilla.redhat.com/show_bug.cgi?id=2256432 are duplicates? Let's close the latter one.

Oh! Thanks for spotting this. I did that.

File: test/verify/check-storage-lvm2 (hunk 1 above)
Comment (mvollmer) [reply test/verify/check-storage-lvm2:818]: Yep.

**Review by martinpitt — APPROVED** [review]
Looks good, thanks! The pixel failures are fallout from tasks container refresh (see #19944), the third failure looks like an unrelated flake.

**Review by allisonkarlitskaya — APPROVED** [review]
Let's get this in.

**Pull request merged**